src/mango_selector_text.erl

```diff
@@ -171,11 +171,12 @@ convert(_Path, {Props} = Sel) when length(Props) > 1 ->
 
 
 to_query({op_and, Args}) when is_list(Args) ->
-    Res = ["(", join(<<" AND ">>, lists:map(fun to_query/1, Args)), ")"],
+    Res = ["(", mango_util:join(<<" AND ">>, lists:map(fun to_query/1, Args)),
+        ")"],
```

Comment:
I prefer to avoid continuation lines like this. Assign part of it to a temp variable. You end up with the same number of lines and it becomes a lot easier to read. In this particular case I'd probably pull out the lists:map and assign it to something like `QueryArgs = lists:map(fun to_query/1, Args)`